src/pages/en/comparing-astro-vs-other-tools.md

```diff
@@ -250,13 +250,11 @@ Astro component syntax is a superset of HTML. It was designed to feel familiar t
 | Boolean Attributes           | `autocomplete` === `autocomplete={true}` | `autocomplete` === `autocomplete={true}` |
 | Inline Functions             | `{items.map(item => <li>{item}</li>)}`  | `{items.map(item => <li>{item}</li>)}` |
 | Conditional Rendering             | `{condition &&  <p>text<p>}`  | `{condition &&  <p>text<p>}` |
-| IDE Support                  | [VS Code][code-ext] | Phenomenal |
+| IDE Support                  | [VS Code (incl. Open VSX), Nova](/en/editor-setup) | Phenomenal |
```

Comment:
```suggestion
| IDE Support                  | Available for [VS Code (incl. Open VSX), Nova](/en/editor-setup) | Built-in |
```

Reply:
As the original author of "Phenomenal", I don't mind being a little more explicit now.